ckanext/stcndm/logic/common.py

```diff
@@ -402,10 +434,16 @@ def get_upcoming_releases(context, data_dict):
     start_date = _get_or_bust(data_dict, 'startDate')
     end_date = _get_or_bust(data_dict, 'endDate')
 
-    q = {'q': 'release_date:[{startDate}:00Z TO {endDate}:00Z] AND last_publish_status_code:8'.format(
-        startDate=start_date,
-        endDate=end_date),
-        'rows': 500}
+    q = {
+        'q': (
+            'release_date:[{startDate}:00Z TO {endDate}:00Z] '
+            'AND last_publish_status_code:8'
```

Comment:
We're getting there! This is the entire purpose of the refactoring!

get_last_publish_status will show you the code mappings for this. 8 corresponds to "Verified", which means that a record has been approved for release on a given release date.

The product IDs which have inferable data inside them sure are a thing, though.